pynso/connection.py

```diff
@@ -125,9 +125,6 @@ def head(self, *args: Any, **kwargs: Any) -> None:
     def post(self, *args: Any, **kwargs: Any) -> Optional[JSON]:
         response = self._request(self.session.post, *args, **kwargs)
 
```

Comment:
To address what John was saying, do we want to add an arg on line 128 that if passed in would allow you return data?  The idea being this would allow for default behavior to remain, but instead we could pass a flag in that if true, would return data?

Reply:
Great minds think alike! This is the solution John and I landed on as well - it is now implement as you say.